eng/spotbugs-aggregate-report/pom.xml

```diff
@@ -316,7 +316,7 @@
         <dependency>
           <groupId>com.microsoft.azure</groupId>
           <artifactId>azure-servicebus</artifactId>
-          <version>3.1.0</version> <!-- {x-version-update;com.microsoft.azure:azure-servicebus;current} -->
+          <version>3.1.1</version> <!-- {x-version-update;com.microsoft.azure:azure-servicebus;current} -->
```

Comment:
So the tooling uses XML comments to find the line that it has to replace? Interesting! If you have the source code another mechanism might be to use an attribute on the ```<version></version>``` element. That way it could find the element with a simple XPath query.

Reply:
I agree if it was only XML I had to deal with. Right now it's XML, .MD, .Java and I'm told there's another. different file type coming. I went with the path of least resistance and just treat everything as text.